tests/setup/minio/isMinioInstalled/isMinioInstalledBinaryFileExists.windows.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { describe, it, expect, vi, afterAll } from "vitest";
+import { execSync } from "child_process";
+import * as path from "path";
+import * as fs from "fs";
+import { isMinioInstalled } from "../../../../src/setup/isMinioInstalled";
+
+vi.mock("fs", async (importOriginal) => {
+  const actual = await importOriginal();
+  return {
+    ...(actual as object),
+    existsSync: vi.fn().mockReturnValue(true),
+  };
+});
+vi.mock("child_process", () => ({
+  exec: vi.fn(),
+  execSync: vi.fn(),
+  spawnSync: vi.fn(),
+}));
+vi.mock("os", async (importOriginal) => {
+  const actual = await importOriginal();
+  return {
+    ...(actual as object),
+    platform: vi.fn().mockReturnValue("win32"),
+  };
+});
+vi.mock("path", () => ({ join: vi.fn().mockReturnValue("/home/minio/minio") }));
+
+describe("isMinioInstalled - Binary File Exists", async () => {
+  afterAll(() => {
+    vi.resetAllMocks();
+    vi.resetModules();
+  });
+  it("should return true if minio binary file exists", () => {
+    vi.mocked(execSync).mockImplementation((command: string) => {
+      if (command === "minio --version") {
+        throw new Error("Command not found");
+      }
+      return Buffer.from("");
+    });
+
+    expect(isMinioInstalled()).toBe(true);
+    expect(fs.existsSync).toHaveBeenCalledWith(
+      path.join("/home", ".minio", "minio.exe"),
+    );
+  });
```

Comment:
The test case where the binary does not exist is covered in different test file named as isMinioInstalled.spec.ts to ensure isolation for each tests since we are mocking complete packages.